config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -2523,6 +2523,10 @@
 # DEFAULT: false.
 # prefs.research.collab=true
 
+# To set the default value for message notifications on sites
+# DEFAULT: 0 = NO; 1 = YES.
+ prefs.msg.notification=1
```

Comment:
`0` is the advertised default, but this is setting it to `1` by default. This line should be commented out.

Reply:
This is still an issue.